### DISCUSSION TITLE
announcements

### DISCUSSION THREAD

**Discussion by giscus** [FanzOfLenaZavaroni/fanzoflenazavaroni.github.io#2] [Archives]
# announcements

This is a fan made website and is not endorse by Lena Zavaroni’s family or friends.

https://fanzoflenazavaroni.github.io/announcements

**Comment by FanzOfLenaZavaroni** [comment]
A large system failure has meant I had to do a full reinstall of this website. Sadly it has meant the lost of all visitor comments on this website, I still have all the comments as emails but that doesn't mean I will be able to add them back here.

I will look to see if I can run the comment system in a separate repository so that any system failure of the website's repo. will not effect the comments and see if there is a way to back up the comments. That doesn't mean the comments repo. could not have a failure of its own but keeping them separate seem the best way to go if it can be done.

This will take time if it can be done, as my coding skills are limited plus my on going heath issues with my hands has already limited how much time I can spend updating the website without this additional work behind the scenes.

**Reply by FanzOfLenaZavaroni** [reply]
I have started to recreate the comments as best I can from the email versions of the comments I have.

**Reply by FanzOfLenaZavaroni** [reply]
I have defined the recreated comments as Orphaned Comments and they should be available to view on the original page (maybe via a link) they were made on and are grouped under the Category: [Orphaned Comments](https://fanzoflenazavaroni.github.io/categories/#Orphaned-Comments).

As I can't recreated my comments as GitHub does not email me my own comments, not all comments have been recreated I know this is a shame but those I have not recreated make little sense without the context of my comments.

**Comment by FanzOfLenaZavaroni** [comment]
I have change the way biographies are saved on this website they are now located in a sub-folder of the "post" section with their date-of-birth used as the post date this makes them easier to manage with the "Liquid Template Language" and will now automatically show up on the "Posts" page and will also automatically show up on any related "On This Days" E.G, on their birthday and where applicable on their death day.

It also means they now have a section using their name in the "Categories" section so visitors can view all content related to an individual by going to their name within the "Categories" section.

**Reply by FanzOfLenaZavaroni** [reply]
The changes I made to the biographies section have now been made to the podcast section for the same reasons.

**Comment by FanzOfLenaZavaroni** [comment]
Currently re-doing the way "On This Day" works to deal with multiple dates for a single post.

**Reply by FanzOfLenaZavaroni** [reply]
The updated "OnThisDay" code means that dealing with multiple dates has been made more manageable and if required other details along with the date can be added.
